packages/drivers/local-driver/src/localDocumentStorageService.ts

```diff
@@ -91,12 +93,16 @@ export class LocalDocumentStorageService implements IDocumentStorageService {
 				throw new Error("No versions for the document!");
 			}
 
-			versionId = versions[0].treeId;
+			// Non null asserting here because of the length check above
```

Comment:
We should use the standard comments

Reply:
This will be the final comment so I am not adding a TODO